airbyte-webapp/src/packages/cloud/views/auth/LoginPage/LoginPage.tsx

```diff
@@ -23,7 +23,8 @@ const LoginPageValidationSchema = yup.object().shape({
 const LoginPage: React.FC = () => {
   const { formatMessage } = useIntl();
   const { login } = useAuthService();
-  const { query, replace } = useRouter();
+  const query = useRouterQuery();
```

Comment:
can you be more specific on this? not sure I am following the logic here

Reply:
Those comments all refer to https://github.com/airbytehq/airbyte/pull/14869#discussion_r929166307. I think we should remove the `useRouterQuery` hook completely, and instead use either the native `useParams` (if we need it from params) or a `useQuery` hook (that we'd need to build, since weirdly react-router doesn't include one). Those comments all state if the places earlier using `useRouterQuery` actually will require to pull in the information from the query i.el. `useQuery` or from the params i.e. `useParams`.

Reply:
yes. got you. already fixed